### PR TITLE
data

### PR DESCRIPTION
**Pull request by VadimDrobyazko**
[DEMO LINK](https://vadimdrobyazko.github.io/js_get_data_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const item = document.querySelectorAll('.population');
+const newStorke = [...item]
+  .map(type => type.innerHTML)
+  .map(type => parseInt(type.split(',').join('')));
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/scripts/main.js:6]: one extra iteration
```suggestion
const newStorke = [...item]
  .map(type => parseInt(type.innerHTML.split(',').join('')));
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+const itemTotal = newStorke.reduce((a, b) => a + b);
+const itemAvarage = itemTotal / newStorke.length;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/scripts/main.js:9]:
```suggestion
const total = newStorke.reduce((a, b) => a + b);
const avarage = itemTotal / newStorke.length;
```

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Great job, just a few improvements to make it perfect

Don'TStop!Don'TGiveUp!GIF

**Review by etojeDenys — APPROVED** [review]
LGTM

**Pull request closed without merging**